plugins/renovate-backend-module-runtime-s3/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@secustor/backstage-plugin-renovate-backend-module-runtime-s3",
+  "description": "The runtime-s3 backend module for the renovate plugin.",
+  "version": "0.0.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "LGPL-3.0-or-later",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module",
+    "pluginId": "renovate",
+    "pluginPackage": "@secustor/backstage-plugin-renovate-backend"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/secustor/backstage-plugins.git",
+    "directory": "plugins/renovate-backend-module-runtime-s3"
+  },
+  "bugs": {
+    "url": "https://github.com/secustor/backstage-plugins/issues"
+  },
+  "homepage": "https://github.com/secustor/backstage-plugins",
+  "keywords": [
+    "renovate",
+    "backstage"
+  ],
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@aws-sdk/client-s3": "3.787.0",
+    "@aws-sdk/credential-provider-node": "3.787.0",
```

```suggestion
    "@aws-sdk/client-s3": "^3.787.0",
    "@aws-sdk/credential-provider-node": "^3.787.0",
```
Use ranges so no users can use newer versions, as long they are not breaking.